packages/lab/src/combo-box-next/useComboBox.tsx

```diff
@@ -1,28 +1,40 @@
-import { FocusEvent, KeyboardEvent, SyntheticEvent, useState } from "react";
+import { FocusEvent, KeyboardEvent, SyntheticEvent } from "react";
 import { useList, UseListProps } from "../list-next/useList";
 import { useComboboxPortal, UseComboBoxPortalProps } from "./useComboboxPortal";
+import { useControlled } from "@salt-ds/core";
 
 interface UseComboBoxProps {
+  value?: string;
+  defaultValue?: string;
   onBlur?: (event: FocusEvent<HTMLInputElement>) => void;
   onFocus?: (event: FocusEvent<HTMLInputElement>) => void;
   onMouseOver?: (event: SyntheticEvent) => void;
   onKeyDown?: (event: KeyboardEvent<HTMLInputElement>) => void;
+  onSelect?: (
+    event: SyntheticEvent<HTMLInputElement>,
+    data: { value?: string }
+  ) => void;
   PortalProps?: UseComboBoxPortalProps;
-  listProps: UseListProps;
+  listHookProps: UseListProps;
```

Comment:
i'd call this useListHookProps or useListProps, in case more hooks are created for list